pkg/rpctest/rpctest.go

```diff
@@ -0,0 +1,17 @@
+package rpctest
```

Comment:
I added this so I could create an OutputWriter, which is an argument in many functions. My intention is not that any binary other than tests will ever import this package.

I'm open to ideas, for doing this better.

Reply:
I've now removed this, and rpctest was added in https://github.com/testground/testground/pull/1015 instead.